### ISSUE TITLE
Ask AI getting unusable.  Why popup so small and hide output?

### ISSUE THREAD

**Issue by corepay** [supabase/supabase#22714]
# Improve documentation

## Describe the problem

Supabase AI is great in helping me work and reson through what I need to accomplish but I get only a small view of response, requiring a lot of scroll and the worst part is the block "Supabase AI is experimental and may produce incorrect answers. Always verify the output before executing." sits on top of the response text requiring me to copy/paste the last part of the results in anpther doc to see what it says.  It interrupts workflow - once or twice not bad but after doing this a dozen times gets annoying.

You got a great AI feature.  I do not understand the reason why you would make it so inaccessible.

## Describe the improvement

Make the ASK AI popup full screen,  remove the "Supabase AI is experimental and may produce incorrect answers." or at least allow my to close it or scroll it out or hide so I can see the full output.

Always verify the output before executing.
## Additional context

![image](https://github.com/supabase/supabase/assets/48992923/05a18c9a-c999-4850-8e40-e0f79bcc56a8)

**Comment by encima** [comment]
Thanks for the suggestion! I will let someone from the Frontend team respond and we also accept PRs!

**Comment by Hallidayo** [comment]
Hi @corepay - When you hover over the code example is their a copy icon in the top right hand corner? This should allow you to copy the code block without having to manually highlight the code block.

**Comment by corepay** [comment]
Yes this is a good feature but the AI usually provides some valulable supplemental information  and/or watchouts on my question after the code block that is always hidden by the banner.  That was what I was referring to - in order to see this supplemental info I need to copy/paste into another doc to see it.  Here is another screen shot as example that has been scrolled down as far as it allows:

![image](https://github.com/supabase/supabase/assets/48992923/c33c494d-d790-4799-a61a-be3a9cbe803c)
